init-tools.cmd

```diff
@@ -77,6 +77,9 @@ if %UPDATE_CLI_ERRORLEVEL% GTR 1 (
   exit /b %UPDATE_CLI_ERRORLEVEL%
 )
 
+echo Copying supplemental overrides from Tools-Override.
+copy %~dp0Tools-Override\* %~dp0Tools
```

Comment:
Yeah, it's just a temporary thing so I can hack on stuff without touching buildtools. We'll eventually get all that stuff directly into buildtools.